### DISCUSSION TITLE
RAW Sql with include property

### DISCUSSION THREAD

**Discussion by muhammetakkus** [sequelize/sequelize#15276] [Help & Questions]
how can i use raw sql with include property?
I am using this sql statement 

`const [results, metadata] = await sequelize.query("SELECT * from orders where id IN (SELECT order_id from order_details where product_id IN (SELECT product_id from terminal_relations where terminal_id = 5) and is_done = 0 )");`
but i want to include OrderDetail model/table

like this 
`
await Order.findAll({
            include: OrderDetail
        });
`

sequelize.query options dosen't have include property

**Comment by ephys** [comment]
`sequelize.query` only runs SQL. It doesn't have any option such as `include` that could be used to build SQL. You'd need to write the include SQL yourself if you want to use `sequelize.query`
